manifests/apache_exporter.pp

```diff
@@ -100,6 +100,7 @@
   String[1] $config_mode                                             = $prometheus::config_mode,
   String[1] $arch                                                    = $prometheus::real_arch,
   Stdlib::Absolutepath $bin_dir                                      = $prometheus::bin_dir,
+  String $scrape_job_name                                            = 'apache',
```

Comment:
please enforce the minimal string length here as well
```suggestion
  String[1] $scrape_job_name                                         = 'apache',
```

Reply:
you probably need to add the `scrape_job_name`, `scrape_port` and `scrape_port` to all exporters?

Reply:
> you probably need to add the scrape_job_name, scrape_port and scrape_port to all exporters?

i've been just scratching an itch so far. I'm not very familiar with the other exporters so I don't know what port or name name they usually use... so i guess this could be an exercise for the reader? the point of the PR is to kickstart the collection, if there are bits missing i don't think it's that big of a deal for a first phase...